### PR TITLE
Update offline bundle instructions from CSI to CSM Operator

### PR DESCRIPTION
**Pull request by KerryKovacevic**
# Description
Updated documentation for offline bundle installation from referencing CSI Operator to CSM Operator.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1081|

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by EvgenyUglov — APPROVED** [review]

**Comment by shaynafinocchiaro** [comment]
Can you create a github issue and link it here?

**Review by shanmydell — CHANGES_REQUESTED** [review]

**Comment by KerryKovacevic** [comment]
@shanmydell Based on the comments left on my other driver PRs, I am assuming your request was meant to refer to "DRIVERNAME" in the script. I have updated that here and will update for the rest of the repos as well.

**Review by shanmydell — DISMISSED** [review]

**Review by delldubey — APPROVED** [review]
lgtm

**Review by PeresKereotubo — DISMISSED** [review]

File: dell-csi-helm-installer/csi-offline-bundle.md [hunk 1]
```diff
@@ -12,8 +11,8 @@ This includes the following drivers:
 * [PowerStore](https://github.com/dell/csi-powerstore)
 * [Unity](https://github.com/dell/csi-unity)
 
-As well as the Dell CSI Operator
-* [Dell CSI Operator](https://github.com/dell/dell-csi-operator)
+The `csm-offline-bundle.sh` script can be used to create a package for offline installation of Dell CSI storage providers for deployment via the CSM Operator.
```
Comment (gallacher) [inline dell-csi-helm-installer/csi-offline-bundle.md:14]: "... to create a package for **the** offline installation of ... "

**Review by adarsh-dell — APPROVED** [review]
LGTM.

**Pull request merged**